update-affiliated/update_astropy_helpers.py

```diff
@@ -21,6 +21,13 @@
 except IndexError:
     raise IndexError("Please specify the helpers version as argument")
 
+
+if HELPERS_TAG.startswith():
```

Comment:
Is there something missing in the  ``startswith()``?

Reply:
fixed